manifests/ca.pp

```diff
@@ -104,6 +107,7 @@
   $key_name     = '',
   $key_ou       = '',
   $tls_auth     = false,
+  $easyrsa_source = $::openvpn::params::easyrsa_source,
```

Comment:
That way it will take the current values from `params.pp` but allow you to override it in the class call or over hiera. I don't see how this could be done in `params.pp` as there is a lot of conditional logic for different operating systems. Additionally this is a pretty common way of overriding values or did i misunderstand your request?